fog/view/server/src/server.rs

```diff
@@ -102,24 +102,23 @@ where
                 Arc::new(AnonymousAuthenticator::default())
             };
 
-        let fog_view_service = view_grpc::create_fog_view_api(FogViewService::new(
-            enclave.clone(),
-            Arc::new(recovery_db),
-            db_poll_thread.get_shared_state(),
-            client_authenticator,
-            config.client_listen_uri.clone(),
-            sharding_strategy,
-            logger.clone(),
-        ));
-        log::debug!(logger, "Constructed View GRPC Service");
-
         // Health check service
         let health_service =
             mc_util_grpc::HealthService::new(Some(readiness_indicator.into()), logger.clone())
                 .into_service();
 
         let server_builder = match config.client_listen_uri {
             ClientListenUri::ClientFacing(ref fog_view_uri) => {
+                let fog_view_service = view_grpc::create_fog_view_api(FogViewService::new(
+                    enclave.clone(),
+                    Arc::new(recovery_db),
+                    db_poll_thread.get_shared_state(),
+                    client_authenticator,
+                    config.client_listen_uri.clone(),
+                    sharding_strategy,
+                    logger.clone(),
+                ));
+                log::debug!(logger, "Constructed client-facing View GRPC Service");
```

Comment:
:wrench: this debug message directly before the info message feels odd, perhaps only the info one is needed?  
not sure if we have debug only logs separate from info only logs
```suggestion
```

Reply:
This is how it was before and I think it's sometimes used in debug logging.We do have debug only logs when debugging, if that's what you're asking. Therefore, I'd opt to just keep it.